kafka-inmemory/resources/kafka-headless-service.yaml

```diff
@@ -2,6 +2,8 @@ apiVersion: v1
 kind: Service
 metadata:
   name: kafka-headless
+  annotations:
+    service.alpha.kubernetes.io/tolerate-unready-endpoints: "true"
```

Comment:
Since which version of k8s is this available?

Reply:
I don't know ... it works at least from Kubernetes 1.6.

Reply:
@tombentley It was added in Kubernetes 1.3.